docs/quiz-toolkit/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Quiz Toolkit",
```

Comment:
Instead of 'Quiz Toolkit' can we create a 'Tutorials' or 'Advanced Tutorials' label and place it at the end just before 'mecsimcalc Library'

Then it would be nice if we could combine all the Quiz Toolkit into one page.